src/rvsdg/egglog_optimizer/mod.rs

```diff
@@ -31,7 +31,7 @@ pub fn rvsdg_egglog_schedule() -> String {
     // they take many iterations.
 
     "(run-schedule
-        (repeat 5 (saturate fast-analyses)
+        (repeat 7 (saturate fast-analyses)
```

Comment:
@oflatt is it ok to bump this?

Reply:
Instead, could you please add your interval rules to the `fast-analyses` ruleset?